src/Package.php

```diff
@@ -84,6 +84,50 @@ public function setMessages(array $messages)
 
     /**
      *
+     * Adds one message for this package.
+     *
+     * @param string $key the key of the message
+     *
+     * @param string $message the actual message
+     *
+     * @return void
+     *
+     */
+    public function addMessage($key, $message)
+    {
+        $this->messages[$key] = $message;
+    }
+
+    /**
+     *
+     * Adds new messages for this package.
+     *
+     * @param array $messages The messages to add in this package.
+     *
+     * @return void
+     *
+     */
+    public function addMessages($messages)
+    {
+        $this->messages = array_merge($this->messages, $messages);
+    }
+
+    /**
+     *
+     * Removes one message for this package.
+     *
+     * @param string $key the key of the message to remove
+     *
+     * @return void
+     *
+     */
+    public function removeMessage($key)
```

Comment:
Honestly i did this by force of habit when i do mutable stuff. It's true that it might not ever be necessary in real scenario.